saleor/graphql/order/types.py

```diff
@@ -299,6 +302,10 @@ def resolve_translated_product_name(root: models.OrderLine, _info):
     def resolve_translated_variant_name(root: models.OrderLine, _info):
         return root.translated_variant_name
 
+    @staticmethod
+    def resolve_allocations(root: models.OrderLine, _info):
```

Comment:
Permissions?
This field should be visible to staff users who create fulfillment.

Reply:
I would say we should require either `MANGE_ORDERS` or `MANAGE_PRODUCTS`:
```
@one_of_permissions_required([ProductPermissions.MANAGE_PRODUCTS, OrderPermissions.MANAGE_ORDERS])
```